imagetest/test_suites/oslogin/ssh_test.go

```diff
@@ -156,7 +161,7 @@ func TestAgent(t *testing.T) {
 		t.Fatalf("Error changing metadata: %v", err)
 	}
 	// Give the API time to update.
-	time.Sleep(time.Second * 5)
+	time.Sleep(time.Second * 4)
```

Comment:
I see you are shifting some of these sleep times, why?

Reply:
I've tried reducing the time to see if we can get away without it. There was no difference between 4s and 5s, so I went with the lower one to save a few seconds per test run.